### ISSUE TITLE
urwid.display_common.AttrSpecError: foreground/background ('bold, #6d0'/'g11') require more colors than have been specified (16).

### ISSUE THREAD

**Issue by duckdalbe** [adbenitez/deltachat-cursed#81]
I installed from the "master"-branch using `pip install -U --user "git+https://github.com/adbenitez/deltachat-cursed@master"`.

Starting `curseddelta` gives me this error:

```
Traceback (most recent call last):
  File "/home/user/.local/bin/curseddelta", line 8, in <module>
    sys.exit(main())
  File "/home/user/.local/lib/python3.10/site-packages/deltachat_cursed/util.py", line 188, in wrapper
    return func(*args, **kwargs)
  File "/home/user/.local/lib/python3.10/site-packages/deltachat_cursed/main.py", line 67, in main
    start_ui(args)
  File "/home/user/.local/lib/python3.10/site-packages/deltachat_cursed/main.py", line 329, in start_ui
    app = Application(args.acct, args.cfg, keymap, theme, args.logger)
  File "/home/user/.local/lib/python3.10/site-packages/deltachat_cursed/application.py", line 42, in __init__
    self.main_loop: urwid.MainLoop = urwid.MainLoop(
  File "/usr/lib/python3.10/site-packages/urwid/main_loop.py", line 118, in __init__
    screen.register_palette(palette)
  File "/usr/lib/python3.10/site-packages/urwid/display_common.py", line 856, in register_palette
    self.register_palette_entry(*item)
  File "/usr/lib/python3.10/site-packages/urwid/display_common.py", line 923, in register_palette_entry
    basic = AttrSpec(foreground, background, 16)
  File "/usr/lib/python3.10/site-packages/urwid/display_common.py", line 540, in __init__
    raise AttrSpecError(('foreground/background (%s/%s) require ' +
urwid.display_common.AttrSpecError: foreground/background ('bold, #6d0'/'g11') require more colors than have been specified (16).
```

As far as I can tell, `urwid` is used from my system installation, which provides v2.1.2.
I have no idea how to further debug this situation. If I should provide you with more details, please tell me.

This is on an up-to-date Archlinux.

Full installation log:

```
Collecting git+https://github.com/adbenitez/deltachat-cursed@master
  Cloning https://github.com/adbenitez/deltachat-cursed (to revision master) to /tmp/pip-req-build-6jl9r917
  Running command git clone --filter=blob:none --quiet https://github.com/adbenitez/deltachat-cursed /tmp/pip-req-build-6jl9r917
  Resolved https://github.com/adbenitez/deltachat-cursed to commit b4f196dce5d8478da952d7868b4b8fd1e2a71ad8
  Preparing metadata (setup.py) ... done
Collecting deltachat>=1.79.0
  Using cached deltachat-1.93.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (10.4 MB)
Requirement already satisfied: urwid>=2.1.2 in /usr/lib/python3.10/site-packages (from deltachat-cursed==0.7.2.dev4+gb4f196d) (2.1.2)
Collecting urwid-readline>=0.13
  Using cached urwid_readline-0.13-py3-none-any.whl
Collecting notify-py>=0.3.3
  Using cached notify_py-0.3.3-py3-none-any.whl (649 kB)
Collecting emoji>=2.0.0
  Using cached emoji-2.0.0-py3-none-any.whl
Collecting imap-tools
  Using cached imap_tools-0.57.0-py3-none-any.whl (31 kB)
Requirement already satisfied: requests in /usr/lib/python3.10/site-packages (from deltachat>=1.79.0->deltachat-cursed==0.7.2.dev4+gb4f196d) (2.28.1)
Collecting pluggy
  Using cached pluggy-1.0.0-py2.py3-none-any.whl (13 kB)
Requirement already satisfied: cffi>=1.0.0 in /usr/lib/python3.10/site-packages (from deltachat>=1.79.0->deltachat-cursed==0.7.2.dev4+gb4f196d) (1.15.1)
Collecting loguru<0.6.0,>=0.5.3
  Using cached loguru-0.5.3-py3-none-any.whl (57 kB)
Requirement already satisfied: jeepney in /usr/lib/python3.10/site-packages (from notify-py>=0.3.3->deltachat-cursed==0.7.2.dev4+gb4f196d) (0.8.0)
Requirement already satisfied: pycparser in /usr/lib/python3.10/site-packages (from cffi>=1.0.0->deltachat>=1.79.0->deltachat-cursed==0.7.2.dev4+gb4f196d) (2.21)
Requirement already satisfied: idna<4,>=2.5 in /usr/lib/python3.10/site-packages (from requests->deltachat>=1.79.0->deltachat-cursed==0.7.2.dev4+gb4f196d) (3.3)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/lib/python3.10/site-packages (from requests->deltachat>=1.79.0->deltachat-cursed==0.7.2.dev4+gb4f196d) (1.26.12)
Building wheels for collected packages: deltachat-cursed
  Building wheel for deltachat-cursed (setup.py) ... done
  Created wheel for deltachat-cursed: filename=deltachat_cursed-0.7.2.dev4+gb4f196d-py3-none-any.whl size=39102 sha256=664ecdc1b521bf5ec300f4ed56935c4bdcc4fb780c2fc5d9f6d07075f0cd023f
  Stored in directory: /tmp/pip-ephem-wheel-cache-j3yyctls/wheels/99/81/53/0f5e04f3eff975ccf2fecce4e3a046b1b786a37ca9dbbaa401
Successfully built deltachat-cursed
Installing collected packages: imap-tools, emoji, urwid-readline, pluggy, loguru, notify-py, deltachat, deltachat-cursed
Successfully installed deltachat-1.93.0 deltachat-cursed-0.7.2.dev4+gb4f196d emoji-2.0.0 imap-tools-0.57.0 loguru-0.5.3 notify-py-0.3.3 pluggy-1.0.0 urwid-readline-0.13
```

**Comment by adbenitez** [comment]
hi @duckdalbe, sorry for the late reply, I am a bit busy at the moment with other deltachat-related projects, but will try to reproduce,

I use Cursed Delta every day on my machine and it is working fine, will try a clean install,
thanks for reporting the issue!

**Comment by jose1711** [comment]
@duckdalbe Try the following:
```
mv ~/.curseddelta/theme.json ~/.curseddelta/theme.json.bak
```
Issue is likely caused by incompatibility between this and older version's format.

**Comment by adbenitez** [comment]
@jose1711 thanks for pointing that out! that might be the issue, 
I think the app should be fixed to check the file content and fallback to default properties if something is wrong

**Comment by adbenitez** [comment]
fixed in 7fe75ef
